samcli/hook_packages/terraform/copy_terraform_built_artifacts.py

```diff
@@ -51,6 +51,14 @@
 import zipfile
 import logging
 
+try:
+    # this case will work only while executing the copy terraform command from the makefile
+    from zip import unzip  # type: ignore
+except ImportError:
+    # this way to fix the hidden import unit testing
+    from samcli.local.lambdafn.zip import unzip
```

Comment:
why do we use something different for unit tests than for the actual use? Doesn't this remove the confidence we can have in our unit tests?

Reply:
The issue is the script will run in  a different way than we do in our unit testing (the hidden import test case) .. let me check with Wing how can we remove it from this test case.

Reply:
Which unit test does it break?

Reply:
my instinct is we should fix the test

Reply:
I skipped the `samcli.hook_packages.terraform.copy_terraform_built_artifacts` from the `test_import_should_succeed_for_a_defined_hidden_package` test case, as this script will be only executed as a standalone script and does not depend on any non standard modules